### ISSUE TITLE
Spotlight search does not look correct in high contrast mode

### ISSUE THREAD

**Issue by langleyd** [element-hq/element-web#34991]
### Steps to reproduce

1. Turn on high contrast
2. Go to spotlight and search for something

See things like this

<img width="421" height="70" alt="Image" src="https://github.com/user-attachments/assets/3157c1b9-dbeb-4cc6-a4bd-84f073063e15" />

<br>
<img width="439" height="72" alt="Image" src="https://github.com/user-attachments/assets/93ec8295-ccc8-48da-aa54-399892281b9b" />

<br>
<img width="241" height="70" alt="Image" src="https://github.com/user-attachments/assets/1960cd49-2400-49e9-a755-986288a82249" />
<br>

<img width="442" height="72" alt="Image" src="https://github.com/user-attachments/assets/4b18c2b8-5715-445d-a02e-a256adb9b3ef" />

#### What did you expect?
The alias text is high contrast for both hovered and uncovered.
The notification looks appropriate and you can read counts.

#### What happened instead?
The alias text is only high contrast for hovered.
The notification looks appropriate and you cannot read counts.

### Operating system

macOS

### Browser information

Chrome

### URL for webapp

develop.element.io

### Application version

Element version: a3dad01571f0-js-f6a2e3335669 Crypto version: Rust SDK 0.18.0 (e5f8295), Vodozemac 0.10.0

### Will you send logs?

No
